configure.ac

```diff
@@ -49,21 +49,17 @@ AC_PREFIX_DEFAULT([ucx_prefix])
 top_top_srcdir=$srcdir
 AC_SUBST(top_top_srcdir)
 
-RPM_RELEASE=1
-BUILD_TS=ts
 MAJOR_VERSION=ucx_ver_major
 MINOR_VERSION=ucx_ver_minor
-MICRO_VERSION=scm_r
-REV_COUNT=revcount
-VERSION=$MAJOR_VERSION.$MINOR_VERSION.$REV_COUNT
+PATCH_VERSION=ucx_ver_patch
+SCM_VERSION=scm_r
+VERSION=$MAJOR_VERSION.$MINOR_VERSION.$PATCH_VERSION.$SCM_VERSION
```

Comment:
@shamisp commented yesterday that we do not want the $SCM_VERSION in rpm name.
1. Is this prohibited by Fedora?
1. Is the a better way to add git version info to the rpm file?